src/components/Documentation/Layout/index.tsx

```diff
@@ -18,6 +19,10 @@ const Layout: LayoutComponent = ({ children, ...restProps }) => {
 
   const toggleMenu = useCallback(() => setIsMenuOpen(!isMenuOpen), [isMenuOpen])
 
+  useEffect(() => {
+    focusElementWithHotkey('#doc-search', '/')
```

Comment:
We need to be able to close this event listener when the `Documentation/Layout` component unmounts. Otherwise this function will keep listening for keystrokes even if the user moves to a different page. One way I can think of to accomplish this is to have`focusElementWithHotkey` return a function that closes the event listener, so the `useEffect` hook can return it.
Example:
```
// in Layout
  useEffect(() => {
    const closeEventListener = focusElementWithHotkey('#doc-search', '/')
    return closeEventListener
  }, [])

...

// in focusElementWithHotkey

export const focusElementWithHotkey = (
  target: string,
  hotkey: string
): (() => void) => {
  const func = (e: KeyboardEvent): void => {
    // place code for keyboard event here
  }
  document.addEventListener('keydown', func)
  return (): void => document.removeEventListener('keydown', func)
}
```

Reply:
Thanks for the feedback! I've addressed this issue in 44a3221